Line 47 added description for ssh -L not -1 (#11)
compsys-progtools/fall2024: notes/2024-10-24.md

+++{"lesson_part": "main","type":"output"}

> When it logs in it looks like this and requires you to change your password.  They configure it with a default and with it past expired. Please note the cpmmand is ssh -l or "L" not the number 1!

Reviewer: ```suggestion
When it logs in it looks like this and requires you to change your password.  They configure it with a default and with it past expired. Please note the command  `ssh -l`, includes a lowercase  "L" not the number 1!
```
